packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -179,7 +179,9 @@ export class TestCase {
     this.assert.strictEqual(newSnapshot.length, oldSnapshot.length, 'Same number of nodes');
 
     for (let i = 0; i < oldSnapshot.length; i++) {
-      this.assertSameNode(newSnapshot[i], oldSnapshot[i]);
+      if (!(newSnapshot[i] instanceof TextNode && oldSnapshot[i] instanceof TextNode)) {
+        this.assertSameNode(newSnapshot[i], oldSnapshot[i]);
+      }
```

Comment:
I think this is fine for now as long as we clean it up before merge. I'll try to come up a suggestion. I have a few ideas, but I need to look into the details to know if they would work.

As for your question, this is expected. The content inside a triple curlies are parsed and populated by the browser. If the source string changes, we have to wipe out the existing nodes and have the browser re-create them. So this is currently just a definciency in the test abstraction, not a bug in the implementation.